buildkite/src/Jobs/Test/SnarkyJSTest.dhall

```diff
@@ -20,7 +20,7 @@ Pipeline.build
         dirtyWhen = [
           S.strictlyStart (S.contains "buildkite/src/Jobs/Test/SnarkyJSTest"),
           S.strictlyStart (S.contains "buildkite/scripts/test-snarkyjs-bindings.sh"),
-          S.strictlyStart (S.contains "src")
+          S.strictlyStart (S.contains "src/lib")
```

Comment:
yeah I also think snarkyjs doesn't currently, and shouldn't ever have, dependencies outside "src/lib". To hope is to avoid running CI on at least some PRs